node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -403,13 +403,14 @@ class NodeRuntime private[node] (
       equivocationsTrackerLogPath = dagStoragePath.resolve("equivocationsTrackerLogPath"),
       equivocationsTrackerCrcPath = dagStoragePath.resolve("equivocationsTrackerCrcPath"),
       checkpointsDirPath = dagStoragePath.resolve("checkpointsDirPath"),
+      blockNumberIndexPath = dagStoragePath.resolve("blockNumberIndexPath"),
+      mapSize = 100L * 1024L * 1024L * 4096L,
```

Comment:
0,4 TB?

Reply:
I was reading this as 4GB for some reason. Apparently we are [trying](https://github.com/rchain/rchain/blob/48e8ad2f34e7ef2ea2a2d31e1c133836b01e7d12/node/src/main/scala/coop/rchain/node/NodeRuntime.scala#L390) to allocate 0,4 TB for block store ATM. I changed it to 8 GB for both block store and DAG storage.